actions.py

```diff
@@ -0,0 +1,168 @@
+        check = await self.is_inCall(ctx)
+        if check and self.audio_queue:
+            ctx.voice_client.stop()
+            await ctx.send('Song has been skipped!')
+        elif check and ctx.voice_client.is_playing():
+            ctx.voice_client.stop()
+            await ctx.send('Song has been skipped!')
+        elif not ctx.voice_client.is_playing():
+            await ctx.send(f'{ctx.author.name}, boss there is nothing queued to be skipped')
+
+    async def queue(self, ctx: commands.Context):
+        if await self.is_inCall(ctx):
+            songQueue = "Queue\n --------- \n" + \
+                '\n'.join([f"{index}: {song['title']}" for index,
+                          song in enumerate(self.audio_queue, start=1)])
+            await ctx.send(songQueue)
+
+    async def remove(self, ctx: commands.Context, index):
+        if await self.is_inCall(ctx) and index.isnumeric() and int(index) <= len(self.audio_queue):
+            async with self.queue_lock:
+                song = self.audio_queue[int(index) - 1]['title']
+                del self.audio_queue[int(index) - 1]
+                await ctx.send(f'{song} has been removed from queue!')
+    
+    async def clear(self, ctx: commands.Context):
+        if await self.is_inCall(ctx):
+            async with self.queue_lock:
+                self.audio_queue.clear()
+                await ctx.send("Queue has been cleared!")
+
+    async def jump(self, ctx: commands.Context, index):
+        if await self.is_inCall(ctx) and index.isnumeric() and 1 < int(index) <= len(self.audio_queue):
+            async with self.queue_lock:
+                song = self.audio_queue[int(index) - 1]
+                del self.audio_queue[int(index) - 1]
+                self.audio_queue.appendleft(song)
+        elif index == '1':
+            await ctx.send("Brother it's already at the top of the queue")
+        else: 
+            await ctx.send("Enter a number that's in the queue dude")
+
+    async def help(self, ctx: commands.context):
+        await ctx.send('''Commands\n ---------------------
+        Every command uses "!" as a prefix
+        !play '<link> or name' you can pass in a youtube link or just type the song name. Sometimes video plays so whatever 
+        song you search add "lyrics" at the end if you're not passing in the yt link
+        !queue returns the all the songs queued up
+        !skip skip the song playing currently
+        !remove <number in queue> Just input the number on the queue list you wanna remove
+        !clear wipe all the songs in the queue .. 
+        !jump <number in queue> If you want to move something to the top of the queue
+        If bot breaks just pm me and I'll take a look when I can, this is still in testing phase
+        You can use the first letter of every command if you're lazy to type i.e : !p...''')
+
+    '''
+       If more than 1 bot is called to channel lol, I'll make this into a command later
```

Comment:
This is for future use, if we ever have multiple bots. I had it incase we wanted to move the bot before to another channel which we couldn't do before. I recently added the functionality for the bot to move channels if nots playing something